### DISCUSSION TITLE
Compact Machine 4 + Pressurized tube

### DISCUSSION THREAD

**Discussion by aaronaustin0** [NillerMedDild/Enigmatica6#3939] [General]
Hello, 

I may be missing something, but I'm noticing that pressurized tubing is not connecting through the compact machine interfaces to the outside world. 
![image](https://user-images.githubusercontent.com/26975565/146949677-8eff6496-2dac-46e9-aa9b-9900eba9ea7f.png)

I'm currently not finding a way to transport gasses between dimensions. Does anyone have any suggestions?

**Comment by Darkere** [comment]
... Quantum Entangloporter

**Reply by MuteTiefling** [reply]
just be aware that there's no way to transport _radioactive_ gasses through dimensions, and this is by design.
